core/src/util/extrude.cpp

```diff
@@ -0,0 +1,82 @@
+#include "extrude.h"
+#include <cmath>
+#include <cstdlib>
+
+namespace Tangram {
+
+Extrude parseExtrudeString(const std::string& _str) {
+
+    // Values specified from the stylesheet are assumed to be meters with no unit suffix
+    float first = 0, second = 0;
+
+    if (_str == "true") {
+        // "true" means use default properties for both heights, we indicate this with NANs
+        return Extrude(NAN, NAN);
+    }
+
+    if (_str == "false") {
+        // "false" means perform no extrusion
+        return Extrude(0, 0);
+    }
+
+    // Parse the first of two possible numbers
+    const char* pos = _str.c_str();
+    char* end = nullptr;
+
+    // Get a float if possible & advance pos to the end of the number
+    first = std::strtof(pos, &end);
+    if (pos == end) {
+        // No numbers found, return zero extrusion
+        return Extrude(0, 0);
+    }
+
+    // Advance and skip the delimiter character
+    pos = end + 1;
```

Comment:
Right, I based this functionality on observed behavior in tangram-play. It would actually be incorrect for us to accept units here, since tangram-js doesn't. 

Looking more closely, the extrusion behavior in tangram-js is... nuanced. 

`10m` behaves like `true`
`blah` behaves like `true`
`[10, 20m]` behaves like `[10, 0]`
`[10m, 20]` behaves like `[0, 20]`

It also seems that `extrude: 10` extrudes from `min_height` to 10 (which results in some inside-out walls). I had expected it to extrude from 0 to 10.

This PR won't precisely match some of these behaviors; I'm not sure that these are all desirable anyway.